dkim/query.go

```diff
@@ -101,6 +101,7 @@ func parsePublicKey(s string) (*queryResult, error) {
 	if p == "" {
 		return nil, permFailError("key revoked")
 	}
+	p = strings.ReplaceAll(p, " ", "")
```

Comment:
I am wondering, what that change is for?

Reply:
For some website, the key string might have extra space (likely copy/paste formatting issue).  Our Dkim verification will fail when parsing key. Gmail etc accepts it. This change will make us more tolerable.